packages/security/src/deviceTrust/checks/DebuggerCheck.ts

```diff
@@ -0,0 +1,22 @@
+import { SecurityCheck } from "../SecurityCheck";
+import { SecurityCheckResult } from "../SecurityCheckResult";
+
+declare let cordova: any;
+
+export class DebuggerCheck implements SecurityCheck {
+    public readonly name = "Is Debugger Check";
+
+    public check(): Promise<SecurityCheckResult> {
+        return new Promise((resolve, reject) => {
+            if (!cordova.plugins.isDebug) {
+                reject(new Error("Could not find plugin isDebug."));
+                return;
+            }
+
+            cordova.plugins.isDebug((passed: boolean) => {
+                const result: SecurityCheckResult = {name: this.name, passed};
```

Comment:
Based on this description of the result interface in the Android SDK could we make this so that the more secure result is a pass, and the less secure result is a fail? That way all of our SDKs will be the same in their approach.

https://github.com/aerogear/aerogear-android-sdk/blob/master/auth/src/main/java/org/aerogear/mobile/security/SecurityCheckResult.java#L24

I will be updating the rooted check to do the same. Updating it to be `NotRootedCheck`

See (https://github.com/aerogear/aerogear-android-sdk/blob/master/auth/src/main/java/org/aerogear/mobile/security/checks/NoDebuggerCheck.java) for the debugger check.